src/js/PluginActions.js

```diff
@@ -0,0 +1,2 @@
+var {PluginActions} = window["marathonPluginInterface"];
```

Comment:
You mean why having this in an extra file and not exporting it directly in the `main.js` or why exporting this at all? 
Having it in a separate file is about consistency. Every part of plugin SDK lives in a separate and is import/exported within the `main.js`.